languages/csharp/exercises/concept/time/.meta/Example.cs

```diff
@@ -0,0 +1,128 @@
+using System;
+using System.Globalization;
+
+public enum Location
+{
+    NewYork,
+    London,
+    Paris
+}
+
+public enum AlertLevel
+{
+    Early,
+    Standard,
+    Late
+}
+
+public static class Appointment
+{
+    public static DateTime ShowLocalTime(DateTime dt)
+    {
+        return dt.ToLocalTime();
+    }
+
+    public static DateTime Schedule(string appointmentDateDescription, Location location)
+    {
+        TimeZoneInfo tziLocation = TimeZoneInfo.FindSystemTimeZoneById(GetTimeZoneId(location));
+        DateTime dtl = DateTime.Parse(appointmentDateDescription);
+        DateTime dtu = TimeZoneInfo.ConvertTimeToUtc(dtl, tziLocation);
+        return dtu;
+    }
+
+    public static DateTime GetAlertTime(DateTime appointment, AlertLevel alertLevel)
+    {
+        TimeSpan noticePeriod = new TimeSpan();
+
+        switch (alertLevel)
+        {
+            case AlertLevel.Early:
+                noticePeriod = new TimeSpan(1, 0, 0, 0);
+                break;
+            case AlertLevel.Standard:
+                noticePeriod = new TimeSpan(1, 45, 0);
+                break;
+            case AlertLevel.Late:
+                noticePeriod = new TimeSpan(0, 30, 0);
+                break;
+        }
+
+        return appointment - noticePeriod;
+    }
+
+    public static bool HasDaylightSavingChanged(DateTime dt, Location location)
+    {
+        DateTime dtPrevious = dt.AddDays(-7);
+        string tzid = string.Empty;
+        TimeZoneInfo tzi = TimeZoneInfo.FindSystemTimeZoneById(GetTimeZoneId(location));
+        return tzi.IsDaylightSavingTime(dtPrevious) != tzi.IsDaylightSavingTime(dt);
+    }
+
+    public static DateTime NormalizeDateTime(string dtStr, Location location)
+    {
+        try
+        {
+            return DateTime.Parse(dtStr, LocationToCulture(location));
+        }
+        catch (Exception)
+        {
+            return new DateTime();
```

Comment:
The tests use `DateTime.MinValue`, which you could also use here:

```suggestion
            return DateTime.MinValue;
```